moo.js

```diff
@@ -45,10 +46,11 @@
       if (obj.global) throw new Error('RegExp /g flag is implied')
       if (obj.sticky) throw new Error('RegExp /y flag is implied')
       if (obj.multiline) throw new Error('RegExp /m flag is implied')
+      if (obj.unicode) throw new Error('RegExp /u flag is not allowed')
```

Comment:
Will the Unicode flag definitely not work with moo, OOI?

Reply:
It definitely won't work correctly if you try to mix non-unicode and unicode. It might work if every regex uses the unicode flag.